### DISCUSSION TITLE
谷歌网页翻译失效

### DISCUSSION THREAD

**Discussion by aceclz** [chunibyocola/sc-translator-crx#82] [Q&A]
您好！谷歌网页翻译功能失效，求问一下有什么不错的解决方法吗？

**Comment by chunibyocola** [comment]
目前只能通过添加 hosts 规则或者直接使用代理，没有其他好方法了。通过添加 hosts 规则其实不是很稳定，经常会失效，想要稳定使用只能通过代理的方法了。
